src/encrypt.js

```diff
@@ -1,10 +1,22 @@
 const Url = require('url-parse')
 const CryptoJS = require('crypto-js')
+const fs = require('fs')
 import { isUrl, isPatternUrl, isPatterObject, isPattern, isPath } from '../util'
 
+const configData = fs.readFileSync(__dirname + '/config.json', 'utf8')
+const configObject = JSON.parse(configData)
```

Comment:
JSON.parse should be inside try / catch block. there is possibility of circular exception error. can be like below.

```
let moduleConfig = {}
try {
moduleConfig = JSON.parse(configData)
} catch (e) {
}
```

Reply:
Sure will work on it.